docs/source-pytorch/advanced/model_parallel.rst

```diff
@@ -114,9 +118,27 @@ To reach larger parameter sizes and to be memory efficient, we have to shard par
     This is a limitation of Fully Sharded Training that will be resolved in the future.
 
 Enabling Module Sharding for Maximum Memory Efficiency
-""""""""""""""""""""""""""""""""""""""""""""""""""""""
+------------------------------------------------------
+
+Auto Wrapping
```

Comment:
this is the only notable doc update. Rest is just header type update for better sidebar.